---
title: "Meta-knowledgebase of cancer genomic variants"
date: 2018-07-12
layout: default
author: mbaudis
excerpt_separator: <!--more-->
pdf_file_name: 2018-07-12___Wagner_et_al.__A_Meta-Knowledgebase_of_Cancer_Variants__biorXiv.pdf
pdf_file_type: article   # slides poster article
www_links_formatted: '<a href="https://www.biorxiv.org/content/early/2018/07/11/366856" target="_blank">[biorXiv]</a>'  # use this for a quoted, complete HTML link with label 
category: 
  - publications      # publications howto presentation
tags: 
  - article          # article talk poster slides 
---

### A harmonized meta-knowledgebase of clinical interpretations of cancer genomic variants
#### Wagner AH, Walsh B, Mayfield G, Tamborero D, Sonkin D, Krysiak K, Deu Pons J, Duren R, Gao J, McMurry J, Patterson S, Del Vecchio Fitz C, Sezerman OU, Warner J, Rieke DT, Aittokallio T, Cerami E, Ritter D, Schriml LM, Haendel M, Raca G, Madhavan S, Baudis M, Beckmann JS, Dienstmann R, Chakravarty D, Li XS, Mockus SM, Elemento O, Schultz N, Lopez-Bigas N, Lawler M, Goecks J, Griffith M, Griffith OL, and Margolin A
##### bioRxiv. doi:10.1101/366856

<!--  CONTENT  -->



<!--more-->

Precision oncology relies on the accurate discovery and interpretation of genomic variants to enable individualized therapy selection, diagnosis, or prognosis. However, knowledgebases containing clinical interpretations of somatic cancer variants are highly disparate in interpretation content, structure, and supporting primary literature, reducing consistency and impeding consensus when evaluating variants and their relevance in a clinical setting. With the cooperation of experts of the Global Alliance for Genomics and Health (GA4GH) and of six prominent cancer variant knowledgebases, we developed a framework for aggregating and harmonizing variant interpretations to produce a meta-knowledgebase of 12,856 aggregate interpretations covering 3,437 unique variants in 415 genes, 357 diseases, and 791 drugs. We demonstrated large gains in overlapping terms between resources across variants, diseases, and drugs as a result of this harmonization. We subsequently demonstrated improved matching between patients of the GENIE cohort and harmonized interpretations of potential clinical significance, observing an increase from an average of 34% to 57% in aggregate. We developed an open and freely available web interface for exploring the harmonized interpretations from these six knowledgebases at [search.cancervariants.org](http://search.cancervariants.org).

<!-- / CONTENT -->

