---
title: "DNA copy number imbalances in primary cutaneous lymphomas (PCL)"
date: 2019-01-19
layout: default
author: qingyao
excerpt_separator: <!--more-->
pdf_file_name: 2019-01-19___Gug-et-al.__DNA-copy-number-imbalances-in-primary-cutaneous-lymphomas,-PCL__JEADV.pdf
pdf_file_type: article    # slides poster article
www_link:
www_links_formatted:
  - <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jdv.15442" target="_blank">[JEADV article page]</a>
  - <a href="https://europepmc.org/abstract/MED/30659659?fromSearch=singleResult&fromQuery=30659659">[Europe PMC]</a>
category:
  - publications
  - news
tags:
  - article
---

### DNA copy number imbalances in primary cutaneous lymphomas (PCL)
#### Gug G, Huang Q, Chiticariu E, Solovan C and Baudis M  (2019)
##### JEADV, 2019-01-19. https://doi.org/10.1111/jdv.15442

<!--more-->

The article has been published with the _Journal of the European Academy of Dermatology and Venereology_ on January 19, 2019. A corresponding preprint can be accessed through <a href="https://www.biorxiv.org/content/early/2018/09/14/417766" target="_blank">[bioRxiv]</a>.
##### Abstract

**Background**

Cutaneous lymphomas (CL) represent a clinically defined group of extran‐ odal non‐Hodgkin lymphomas harbouring heterogeneous and incompletely delineated molecular aberrations. Over the past decades, molecular stud‐ ies have identified several chromosomal aberrations, but the interpreta‐ tion of individual genomic studies can be challenging.

**Objective**

With a comprehensive meta‐analysis, we aim to delineate genomic alter‐ ations for different types of CL and propose a more accurate classifica‐ tion in line with their various pathogenicity.

**Methods**

We searched PubMed and ISI Web of Knowledge for publications from 1996 to 2016 reporting the investigation of CL for genome‐wide copy number alterations, by means of comparative genomic hybridisation tech‐ niques and whole genome and exome sequencing. We then extracted and re‐mapped the available copy number variation (CNV) data from these publications with the same pipeline and performed clustering and visuali‐ sation to aggregate samples of similar CNV profiles.

**Results**

For 449 samples from 22 publications, copy number variation data was accessible for sample based meta‐analysis. Our findings illustrate struc‐ tural and numerical chromosomal imbalance patterns. Most frequent CNAs were linked to oncogenes or tumour suppressor genes with important roles in the course of the disease.

**Conclusion**

Summary profiles for genomic imbalances, generated from case‐specific data, identified complex genomic imbalances, which could discriminate between different subtypes of CL and promise a more accurate classifi‐ cation. The collected data presented in this study are publicly available through the “Progenetix” online repository.