---
title: "Candidate targets of copy number deletion events across 17 cancer types"
layout: default
authors:
 - '@mbaudis'
 - '@qingyao'
excerpt_separator: <!--more-->
excerpt_link:
pdf_file_name: 2022-06-29___Huang-and-Baudis__Candidate-targets-of-copy-number-deletion-events-across-17-cancer-types__biorXiv.pdf
pdf_file_type: article    # slides poster article
www_link:
www_links_formatted:
  - <a href="https://www.biorxiv.org/content/10.1101/2022.06.29.498080v1" target="_blank">[bioRxiv]</a>
category:
  - publications
  - news
tags:
  - article
  - preprint
  - news
---

### {{page.title}}

#### Huang Q and Baudis M

##### bioRxiv, [doi.org/10.1101/2022.06.29.498080](https://doi.org/10.1101/2022.06.29.498080)

<!--more-->

#### Abstract

Genome variation is the direct cause of cancer and driver of its clonal evolution. While the impact of many point mutations can be evaluated through their modification of individual genomic elements, even single copy number aberrations (CNAs) may encompass hundreds of genes and therefore pose challenges to untangle potentially complex functional effects. However, consistent, recurring and disease-specific patterns in the genome-wide CNA landscape imply that particular CNA may promote cancer type-specific characteristics. Discerning essential cancer-promoting alterations from the inherent co-dependency in CNA would improve the understanding of mechanisms of CNA and provide new insights into cancer biology and potential therapeutic targets.

Here we implement a model using segment breakpoints to discover non-random gene coverage by copy number deletion (CND). With a diverse set of cancer types from multiple resources, this model identified common and cancer type-specific oncogenes and tumor suppressor genes as well as cancer-promoting functional pathways. Confirmed by differential expression analysis of data from corresponding cancer types, the results show that for most cancer types, despite dissimilarity of their CND landscapes, similar canonical pathways are affected. In 25 analyses of 17 cancer types, we have identified 20-170 significant genes by copy deletion, including RB1, PTEN and CDKN2A as the most significantly deleted genes among all cancer types. We have also shown a shared dependence on core pathways for cancer progression in different cancers as well as cancer-type separation by genome-wide significance scores. While this work provides a reference for gene specific significance in many cancers, it chiefly contributes a general framework to derive genome-wide significance and molecular insights in CND profiles with a potential for the analysis of rare cancer types as well as non-coding regions.